### ISSUE TITLE
Support nullable & optional/required marks

### ISSUE THREAD

**Issue by raderio** [springfox/springfox#2825]
Please add support for nullable & optional/required marks.
This is supported by OpenAPI https://swagger.io/docs/specification/data-models/data-types/

We want to represent that a field is non-nullable & optional.
This mean, or we send a data for the field, or just do not send the field in request, null is not allowed.
{"foo": "bar", "baz": true} is allowed
{"baz": true} is allowed, because is optional
{"foo": null, "baz": true} is NOT allowed, because is non-nullable

Basically null has the meaning of "this property existed before, and I want to nullify its value". If there's no value, or the value needs not change (for requests, for example), it just shouldn't be sent.

**Comment by dilipkrish** [comment]
This is an oas 3.0 feature

**Comment by fzyzcjy** [comment]
Hi how can I use `nullable` in springfox? I have annotated with `@Nullable` without any effect
